Terasology/Behaviors
# feature(PathFinding-Behaviour): extract Pathfinding's Behaviors to `PathfindingBehavior` module

This removed code and assets there: https://github.com/Terasology/PathfindingBehaviors

Test:
1. Try common gameplays (JS, LaS)
2. Try CoreGameplay - it should eats less memory - NavGraph was removed from there by default.
